scripts/generator/graphql_generator/csharp/Editor/ShopifyIosPostProcessor.cs.erb

```diff
@@ -0,0 +1,43 @@
+namespace <%= namespace %>.SDK {
+    using UnityEngine;
+    using UnityEditor;
+    using UnityEditor.iOS.Xcode;
+    using System.IO;
+    using System;
+
+    public class ShopifyIosPostProcessor {
+        public static void Process(string buildPath) {
+            string testPath = Path.Combine(buildPath, "Libraries/Plugins/iOS/Shopify/BuyTests/");
+            var testDirectory = new DirectoryInfo(testPath);
+            var project = new ExtendedPBXProject(buildPath);
+
+            try {
+                project.SetFilesInDirectoryToTestTarget(testDirectory);
+            } catch (Exception e) {
+                Debug.Log(e.Message);
+            }
+
+            // Removes default Unity_iPhone_Tests from the Test Target
+            project.RemoveFileFromBuild(project.TestTargetGuid, project.FindFileGuidByProjectPath("Unity-iPhone Tests/Unity_iPhone_Tests.m"));
+            project.SetBuildProperty(project.GetAllTargetGuids(), ExtendedPBXProject.SwiftVersionKey, "3.0");
```

Comment:
Yes, Swift 2.0 and Swift 3.0 differ drastically in terms of naming conventions and other features. With Swift 4.0 coming out in the next WWDC, it would be best that we keep the code base closest to the newest version.